### DISCUSSION TITLE
Missing reference for Shark Liver Oil -> 1-Alkyl-sn-glycerol, and missing human xref for related kinase

### DISCUSSION THREAD

**Discussion by khanspers** [wikipathways/wikipathways-help#31] [General]
Question from Daniela on Slack:

[https://www.wikipathways.org/pathways/WP2533.html](https://www.wikipathways.org/pathways/WP2533.html)
contains "Shark liver oil" as source of 1-Alkyl-sn-glycerol. It seems to be a folk treatment for several diseases, but there is no reference provided. I know we do provide drug mechanisms in several pathways, but this seems a bit weird to me, especially because it looks like this is the only source of this compound. If 1-Alkyl-sn-glycerol needs to be supplemented for this pathway to occur in humans, it would be good to have a reference. But I also didn't find a human protein that would be the alkylglycerol kinase which then phosphorylates this molecule. What are your thoughts on this?

Suggested reference (Denise): https://pubmed.ncbi.nlm.nih.gov/17069352/

**Comment by mkutmon** [comment]
https://www.rhea-db.org/rhea/16940 is this the reaction?

**Comment by khanspers** [comment]
Shark liver oil: https://www.wikidata.org/wiki/Q4380845

**Comment by khanspers** [comment]
There has been no response from the author. I added the Wikidata id as an xref for Shar Liver Oil, and also added the literature reference suggested by Denise.
